sdks/python/apache_beam/yaml/yaml_transform.py

```diff
@@ -892,6 +892,20 @@ def ensure_config(spec):
   return spec
 
 
+def process_logging(spec):
+  if 'logging' in spec:
+    log_level = str(spec['logging']).upper()
+    if log_level not in logging.getLevelNamesMapping():
+      raise ValueError(
+          'Invalid logging level set: '
+          f"{identify_object(spec['logging'])}. "
+          'Valid levels are '
+          f'{", ".join(logging.getLevelNamesMapping())}')
+    logging.getLogger().setLevel(log_level)
+  else:
+    _LOGGER.setLevel('INFO')
```

Comment:
This could be surprising, as setting the logging externally would have no effect if logging is not in the spec.